src/components/YAxis.js

```diff
@@ -174,10 +175,21 @@ export default class YAxis extends React.Component {
         let axisGenerator;
         const axis = align === "left" ? axisLeft : axisRight;
         if (this.props.type === "linear" || this.props.type === "power") {
-            if (this.props.height <= 200) {
-                if (this.props.tickCount > 0) {
+            if (this.props.tickCount > 0) {
+                const stepSize = (this.props.max - this.props.min) / (this.props.tickCount - 1);
+                axisGenerator = axis(scale)
+                    .tickValues(range(this.props.min, this.props.max + 1, stepSize))
```

Comment:
Why `this.props.max + 1`? It seems like this could be very significant depending on what the overall scale is?